ciao-controller/api/api.go

```diff
@@ -461,6 +479,41 @@ func addWorkload(c *Context, w http.ResponseWriter, r *http.Request) (Response,
 	return Response{http.StatusCreated, resp}, nil
 }
 
+func listQuotas(c *Context, w http.ResponseWriter, r *http.Request) (Response, error) {
+	vars := mux.Vars(r)
+	tenantID, ok := vars["tenant"]
+
+	if !ok {
+		tenantID = vars["for_tenant"]
+	}
+
+	var resp types.QuotaListResponse
+
+	resp.Quotas = c.ListQuotas(tenantID)
+
+	return Response{http.StatusOK, resp}, nil
+}
+
+func updateQuotas(c *Context, w http.ResponseWriter, r *http.Request) (Response, error) {
+	vars := mux.Vars(r)
+	tenantID := vars["for_tenant"]
+
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		return errorResponse(err), err
+	}
+
+	var req types.QuotaUpdateRequest
+	err = json.Unmarshal(body, &req)
+	if err != nil {
+		return errorResponse(err), err
+	}
+
+	c.UpdateQuotas(tenantID, req.Quotas)
+
+	return Response{http.StatusCreated, nil}, nil
```

Comment:
with http.StatusCreated, you would not want to return an empty response. While it is not *required* to return a response with Created, it is standard practice and you never know what clients are going to expect. Here you need to either return a list of the quotas you updated, or change the response code to Accepted or NoContent. Since Accepted implies you are not done working on it, it's probably not 'acceptable' here :). IMO you should return the list of updated quotas.